PacketDotNet/MiscUtil/Conversion/StringConverter.cs

```diff
@@ -35,6 +35,8 @@ public static string EbcdicToAscii(byte[] data,int offset,int length)
             {
                 try
                 {
+                    if (data[i] == 0x00)
```

Comment:
why skipping over zero/null values? Can you explain why you are continuing here but still processing the next byte? I'm not familiar with ebcdic.